src/instructlab/configuration.py

```diff
@@ -1495,3 +1495,12 @@ def storage_dirs_exist() -> bool:
         DEFAULTS.LOGS_DIR,
     ]
     return all(os.path.exists(dirpath) for dirpath in dirs_to_check)
+
+
+def detect_storage_dirs():
+    if not storage_dirs_exist():
+        click.secho(
+            "Some ilab storage directories do not exist yet. Please run `ilab config init` before continuing.",
+            fg="red",
+        )
+        raise click.exceptions.Exit(1)
```

Comment:
is this handled without a try, except at the caller?

Reply:
yes

Reply:
So need to add `try/except`?  Since `raise click.exceptions.Exit(1)` will exit directly, try to catch error, seems do the duplicate work, exit again.
e.g.
```
try:
    detect_storage_dirs()
except click.exceptions.Exit:
    <exit again?>
```